cypress/basic/cypress.config.js

```diff
@@ -4,9 +4,60 @@ const { cypressConfig } = require('@axe-core/watcher')
 // Get your configuration from environment variables.
 const { API_KEY, SERVER_URL = 'https://axe.deque.com' } = process.env
 
+/**
+ * Configuration Overrides Section
+ *
+ * Set values for any of the 4 fields described below.
+ * Ensure that the global configuration for your account allows changing these values.
+ * If no values are set, configurationOverrides will be undefined and defaults will be used.
+ *
+ * DEVELOPERS: Set your desired values directly here 👇
+ */
+
+/**
+ * Accessibility standard to use when axe core runs
+ * Example: 'WCAG 2.1 AA'
+ * @type {string|undefined}
+ */
+const accessibilityStandard = undefined // Set your value here
+
+/**
+ * Enable axe core best practices rules
+ * @type {boolean|undefined}
+ */
+const bestPractices = undefined // Set to true or false
+
+/**
+ * Enable experimental axe core rules
+ * @type {boolean|undefined}
+ */
+const experimental = undefined // Set to true or false
+
+/**
+ * Version of axe core to use
+ * Example: '4.1.2'
+ * @type {string|undefined}
+ */
+const axeCoreVersion = undefined // Set your value here
+
+// Build configuration overrides object only if at least one setting is defined
+const configurationOverrides =
+  accessibilityStandard ||
+  bestPractices !== undefined ||
+  experimental !== undefined ||
+  axeCoreVersion
+    ? {
+        ...(accessibilityStandard && { accessibilityStandard }),
+        ...(axeCoreVersion && { axeCoreVersion }),
+        ...(bestPractices !== undefined && { bestPractices }),
+        ...(experimental !== undefined && { experimental })
+      }
+    : undefined
```

Comment:
nit - can be simplify

```suggestion
const configurationOverrides = [
  accessibilityStandard,
  bestPractices,
  experimental,
  axeCoreVersion
].some(value => value !== undefined)
  ? { accessibilityStandard, axeCoreVersion, bestPractices, experimental }
  : undefined;

```

Reply:
Thank you for the suggestion. I've implemented a simplified version of your approach, maintaining the classic conditional structure for readability and easier maintenance:

const configurationOverrides = 
  accessibilityStandard !== undefined || 
  bestPractices !== undefined || 
  experimental !== undefined || 
  execCoreVersion !== undefined 
    ? { accessibilityStandard, execCoreVersion, bestPractices, experimental } 
    : undefined;

I opted for this over the .some() method as it provides a more imperative and explicit approach while still eliminating the spread operator. This achieves our goal of only creating configuration overrides when at least one setting is defined, with improved code clarity.